Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcessAsOwner/Handler.cs

```diff
@@ -18,11 +18,11 @@ public Handler(IIdentitiesRepository identitiesRepository, IUserContext userCont
         _userContext = userContext;
     }
 
-    public async Task<IdentityDeletionProcessDTO> Handle(GetDeletionProcessQuery request, CancellationToken cancellationToken)
+    public async Task<IdentityDeletionProcessDetailsDTO> Handle(GetDeletionProcessAsOwnerQuery request, CancellationToken cancellationToken)
```

Comment:
You should return the Overview, and not the details.

Reply:
Changes made.